client/core/core.go

```diff
@@ -2548,7 +2548,8 @@ func (c *Core) Orders(filter *OrderFilter) ([]*Order, error) {
 func (c *Core) coreOrderFromMetaOrder(mOrd *db.MetaOrder) (*Order, error) {
 	corder := coreOrderFromTrade(mOrd.Order, mOrd.MetaData)
 	oid := mOrd.Order.ID()
-	matches, err := c.db.MatchesForOrder(oid)
+	excludeCancels := false // maybe don't include cancel order matches?
+	matches, err := c.db.MatchesForOrder(oid, excludeCancels)
```

Comment:
This is used by `(*Core).Order` and `(*Core).Orders`, and the consumers may be looking at cancel matches too, so I didn't change this result.